lms/djangoapps/instructor/views/api.py

```diff
@@ -2152,6 +2150,51 @@ def rescore_problem(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('instructor')
+@require_post_params(problem_to_reset="problem urlname to reset", score='overriding score')
+@common_exceptions_400
+def override_problem_score(request, course_id):
+    course_id = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
Could you please substitute this with:

```python
course_key = CourseKey.from_string(course_id)
```

It's helpful to differentiate the opaque key vars with the string representation. Also, I believe that `CourseKey.from_string()` should be your general entrypoint for course key parsing. Using `SlashSeparatedCourseKey` was a practice we started earlier during the migration towards Split that I think we never quite excised from the codebase.